### PR TITLE
Specify elixir version in appropriate places

### PR DESCRIPTION
**Pull request by boddhisattva**
Based on the discussion [here](https://github.com/elixir-lang/elixir-lang.github.com/issues/924) we're also renaming the preferred way to find one's currently installed Elixir version.

### CODE REVIEW

File: install.markdown [hunk 1]
```diff
@@ -145,3 +144,7 @@ On **Unix systems**, you need to [find your shell profile file](https://unix.sta
 ```bash
 export PATH="$PATH:/path/to/elixir/bin"
 ```
+
+## Checking the version of Elixir installed
```
Comment (whatyouhide) [inline install.markdown]: Maybe "Checking the installed version of Elixir" could sounds better? 🤔

File: install.markdown [hunk 2]
```diff
@@ -145,3 +144,7 @@ On **Unix systems**, you need to [find your shell profile file](https://unix.sta
...
+
+Once you have Elixir installed, you can check its version by running `elixir --version`
```
Comment (whatyouhide) [inline install.markdown]: Missing trailing period.

**Comment by eksperimental** [comment]
that's really good in term of consistency and lowering to entry barrier as low as possible. thank you @boddhisattva

File: install.markdown (hunk 2 above)
Comment (boddhisattva) [inline install.markdown]: Thanks for pointing that out 👍

File: install.markdown (hunk 1 above)
Comment (boddhisattva) [inline install.markdown]: Sure, thanks :).

**Comment by boddhisattva** [comment]
Thanks for that perspective [here](https://github.com/elixir-lang/elixir-lang.github.com/issues/924#issuecomment-272995230) @eksperimental :). 

@whatyouhide I've made the changes based on your feedback, kindly check.

**Comment by whatyouhide** [comment]
Thanks a lot @boddhisattva! 💟

**Comment by boddhisattva** [comment]
Thank you Jose, Eksperimental & Andrea. It was a very nice & encouraging experience working with all of you on this PR :).

**Pull request merged**